src/Components/benchmarkapps/Wasm.Performance/TestApp/Pages/Json.razor

```diff
@@ -55,12 +58,18 @@
     void SerializeLarge()
         => serializedValue = JsonSerializer.Serialize(largeOrgChart);
 
+    void SerializeLargeSourceGen()
```

Comment:
Since these are benchmarks, is it worthwhile to also track the fastest perf here which would be:

```cs
using MemoryStream ms = new();
using Utf8JsonWriter writer = new(ms);
personJsonContext.Person.Serialize!(writer, largeOrgChart);
writer.Flush();
serializeValue = Encoding.UTF8.GetString(ms.ToArray());
```

Reply:
In general, we'd like these to be good representation for what our users are doing. These benchmarks are much more useful for us to track relative performance improvements / regressions in these user scenarios rather than being absolute best-case numbers.

Reply:
The [Serialize](https://github.com/dotnet/runtime/blob/78579ef0e7283fd70c410a2d2b0fb9dae6c9d191/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfoOfT.cs#L29) method is public based on the assumption that some users would want absolute-best perf and avoid some overhead of the serializer. Granted, some serializer features like max-depth handling would not be provided in this model, but do we not expect a non-trivial amount of users to want best-case perf?